robots/pr-creator/main.go

```diff
@@ -73,6 +74,7 @@ func optionsFromFlags() options {
 	fs.StringVar(&o.branch, "branch", "", "Repo branch to merge into")
 	fs.StringVar(&o.source, "source", "", "The user:branch to merge from")
 
+	fs.BoolVar(&o.allowMods, "allow-mods", true, "Indicates whether maintainers can modify the pull request")
```

Comment:
Please name flags such that they default to false.

AKA instead of naming this `--allow-mods=true` please do something like `--prevent-mods=false`

Reply:
Instead of renaming (which may cause confusion since if will be inverse of corresponding GH API param), should we just _default_ to `false`?

Reply:
The github api is `maintainer_can_modify` so I wouldn't consider that an issue.

Sure if you want to default to `--allow-mods=false` then I'd keep the current name. If you want to default to `--disallow-mods=false` then I'd choose a name more along those lines (I'd consider something like prevent superior to disallow as don't prevent sounds less like a double negative than don't disallow)

Reply:
👍  I will keep the current name and _default_ to `false`. This will also align with `maintainer_can_modify`.